### DISCUSSION TITLE
Cannot read property 'name' of undefined

### DISCUSSION THREAD

**Discussion by wstcyx** [webpack/webpack#18055] [Q&A]
In version 5.90.1 of webpack, an error is reported in FlagDependencyExportsPlugin.js, and the error content is as follows:

/*****/lib/node_modules/@haier/packi/node_modules/webpack/lib/FlagDependencyExportsPlugin.js:189
                                                                                                        name = exportNameOrSpec.name;
                                                                                                                                ^
TypeError: Cannot read property 'name' of undefined
    at mergeExports (/*****/lib/node_modules/@haier/packi/node_modules/webpack/lib/FlagDependencyExportsPlugin.js:189:38)

I printed the exports and indeed found some undefined. However, I do not know what FlagDependencyExportsPlugin.js is used for. Please fix this issue as soon as possible.
![92991707126972_ pic](https://github.com/webpack/webpack/assets/14196041/bac01992-36e7-4b84-aeb9-1e50acb84f24)

**Comment by wstcyx** [comment]
exports [ 'initializeFEStubs' ]
exports [ 'default' ]
exports [ 'default' ]
exports [ undefined ]

**Comment by alexander-akait** [comment]
Please create a minimum reproducible example
